django_outbox_pattern/consumers.py

```diff
@@ -114,7 +114,7 @@ def _subscribe(self, destination, subscribe_id, headers, queue_name=None, dlq=Fa
             queue_name = f"DLQ.{queue_name}"
         headers.update(
             {
-                "exclusive": settings.DAFAULT_EXCLUSIVE_QUEUE,
+                "exclusive": settings.DEFAULT_EXCLUSIVE_QUEUE,
```

Comment:
**issue**: We can add a test to guarantee the variable `headers` has the keys we want, such as `exclusive` with a particular value like `ACME`.